GettingStarted.md

```diff
@@ -1,80 +1,165 @@
+
+5. Add the `DefaultCursor.prefab` (found under HoloToolkit/Input/Prefabs/Cursor) and add that Object to the **InputManager** Cursor parameter (to avoid it being searched for on scene start) 
+
+> **For Hololens**
+> Optionally, if you wish to enable spatial mapping in your scene for HoloLens, you can add the `SpatialMapping.prefab` (found under HoloToolkit/SpatialMapping/Prefabs) to your 'Managers' object.  Be aware that you must also enable `Spatial Perception` Capabilities: `Edit/Project Settings/Player -> Inspector -> Publishing Settings/Capabilities`.
+
+# 5 Building your project 
+The MRTK provides you a quick and easy way to generate your Unity project from a custom window, which can be found under:
+
+`Mixed Reality Toolkit -> Build Window`
+
+![MRTK Build Window](/External/ReadMeImages/MixedRealityEditorBuildWindow.png)
+
+This window offers many quick options to be able to:
+
+* Build your UWP C# solution
+* Build your project APPX
+* (HoloLens) Deploy your project to a remote device
 
 ![Build Window](/External/ReadMeImages/BuildWindow.PNG)
 
-Optionally, you can use the [default build steps if needed](https://docs.unity3d.com/Manual/windowsholographic-startup.html).
+
+
+# 5.1 Running your project for Immersive Headsets
+Unity supports running your Immersive solution direct from the editor **BUT** Only when the Mixed Reality Portal is running.
+> Unity may resolve this in the future and auto-start the portal, but it is better to have it running beforehand!
+
+1. Start the MR Portal
+2. Plug-in and wake up the headset or [start the Simulator](https://developer.microsoft.com/en-us/windows/mixed-reality/using_the_windows_mixed_reality_simulator)
+3. Make sure the "Cliffhouse" is displayed in the Headset
+4. Select the scene you want to run
+5. Hit Play in the Unity Editor
+
+Provided everything was good, you should now see your scene running in the headset/simulator.
+
+Alternatively, either Use the Unity Build options or the MRTK Build window to generate the UWP package and run locally on your machine.
+
+# 5.2 Running your project for HoloLens
+
+ 1. Be sure to plug in your HoloLens via usb.
+ 2. Open the above Build Window: `Mixed Reality Toolkit -> Build Window`.
+ 3. Under `Deploy` be sure to fill out the device **Username** and **Password** fields.
+ 4. In the `Quick Options` section, press the "`Build SLN, Build APPX, then Install`" button to deploy to HoloLens.
+
+> Optionally, you can use the [default build steps if needed](https://docs.unity3d.com/Manual/windowsholographic-startup.html).
 
 # 6. Deploying your HoloLens app using Visual Studio
- 1. Navigate to the Build WInodw: `HoloToolkit -> Build Window`.
- 2. Press `Open SLN`.
+
+ 1. Navigate to the Build Window: `Mixed Reality Toolkit -> Build Window`.
+ 2. Press `Open Project Solution` to open the project in Visual Studio.
  3. Select **x86** in your build configuration options.
```

Comment:
We should probably also include descriptors about changing to `x64` for Immersive Headsets and deploying to `LocalMachine` vs `RemoteDevice`

Reply:
I don't bump to x64 for Immersive, I build for that in x86 as well

Reply:
If you're in the VS solution you can most likely build to any os

Reply:
Will look at adding a "Deploying WMR" section